spec/services/post_status_service_spec.rb

```diff
@@ -228,13 +228,7 @@
       subject.call(
         account,
         text: 'test status update',
-        media_ids: [
-          Fabricate(:media_attachment, account: account),
-          Fabricate(:media_attachment, account: account),
-          Fabricate(:media_attachment, account: account),
-          Fabricate(:media_attachment, account: account),
-          Fabricate(:media_attachment, account: account),
-        ].map(&:id)
+        media_ids: many_attachments(account, 5).map(&:id)
```

Comment:
What about inlining this?

```suggestion
        media_ids: Array.new(5) { Fabricate(:media_attachment, account: account) }.map(&:id)
```